src/Jobs/GenerateSitemapJob.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+namespace Rapidez\Sitemap\Jobs;
+
+use Illuminate\Bus\Queueable;
+use Illuminate\Contracts\Queue\ShouldQueue;
+use Illuminate\Foundation\Bus\Dispatchable;
+use Illuminate\Queue\InteractsWithQueue;
+use Rapidez\Sitemap\Models\Sitemap;
+
+class GenerateSitemapJob implements ShouldQueue
+{
+    use Dispatchable, InteractsWithQueue, Queueable;
+
+    public function __construct(protected int $storeId) {}
+
+    public function handle(): void
+    {
+        Sitemap::getCachedByStoreId($this->storeId);
```

Comment:
If it's already cached this currently does nothing at all.
Should we clear the cache for that store before calling this?

Reply:
Clearing the cache now before the job gets called!